EngineLayer/CrosslinkSearch/CrosslinkSpectralMatch.cs

```diff
@@ -82,17 +85,18 @@ public void ResolveProteinPosAmbiguitiesForXl()
             }
         }
 
-        public static List<int> GetPossibleCrosslinkerModSites(char[] crosslinkerModSites, PeptideWithSetModifications peptide, InitiatorMethionineBehavior initiatorMethionineBehavior, bool CrosslinkAtCleavageSite)
+        public static void GetPossibleCrosslinkerModSites(char[] crosslinkerModSites, PeptideWithSetModifications peptide, InitiatorMethionineBehavior initiatorMethionineBehavior, bool CrosslinkAtCleavageSite, out List<int> possibleXlPositions)
```

Comment:
Why not public List<int> GetPossibleCrosslinkerModSites? Having "out" seems wrong and I think codacy will complain

Reply:
The idea is to create fewer lists in the search engine to free RAM. 
Not sure why codacy complain, I will check.

Reply:
I think you're still creating a list, just inside the method instead of outside. You can use out instead of tuple to return two values. But, you have a void method and are returning only the list. So, I don't see the point.

Reply:
removed the out method.